README.md

```diff
@@ -39,6 +39,9 @@ strapi:
             # type: photos
             # Permalink used to generate the output files (eg. /articles/:id).
             permalink: /photos/:id/
+            # Permalinks defined for different locales
```

Comment:
Do you think we could document it bit more? :) Otherwise I think it looks good.

Reply:
I've added a paragraph to README